### PR TITLE
HIVE-24876: Disable /logconf.jsp page in HS2 web UI if the user doesn…

### PR DESCRIPTION
**Pull request by saihemanth-cloudera**
…'t belong to admin role

### What changes were proposed in this pull request?
Disbale the logger configuration page for non-admin users.

### Why are the changes needed?
Otherwise normal users can flood log files with unrequired information.

### Does this PR introduce _any_ user-facing change?
Yes. If a user needs to access this log config page, he should be configured as admin in hive-site.xml with the config hive.user.in.admin.role property can have comma separated values.
<property>
    <name>hive.user.in.admin.role</name>
    <value>bob,adam</value>
</property>

### How was this patch tested?
Local machine. Remote cluster.

### CODE REVIEW

File: service/src/resources/hive-webapps/hiveserver2/logconf.jsp [hunk 1]
```diff
@@ -132,7 +134,10 @@
 
                 <button id="log-level-submit" type="button" class="btn btn-primary">Submit</button>
             </form>
-
+            <% } else { %>
+                <p>Cannot configure logging rules unless you user <%= hiveSession.getUserName() %></p>
```
Comment (yongzhi) [inline service/src/resources/hive-webapps/hiveserver2/logconf.jsp:138]: Not quite understand what the sentence means? unless you are an admin user ?

File: service/src/resources/hive-webapps/hiveserver2/logconf.jsp [hunk 2]
```diff
@@ -112,7 +112,9 @@
                     </tbody>
                 </table>
             </div>
-
+            <% Collection<HiveSession> hiveSessions = sessionManager.getSessions();
```
[the remark this answers is not in the document]
Comment (saihemanth-cloudera) [reply service/src/resources/hive-webapps/hiveserver2/logconf.jsp:115]: This HS2 WebUI log interface is common to all the sessions currently. Our goal now is that the log level should only be configured by the users with admin-level privileges. So only the users with admin-level privileges will be seeing this log configuration input.

[the remark this answers is not in the document]
Comment (dengzhhu653) [reply service/src/resources/hive-webapps/hiveserver2/logconf.jsp:115]: Got it, Thanks for the work.

File: service/src/resources/hive-webapps/hiveserver2/logconf.jsp (hunk 1 above)
Comment (saihemanth-cloudera) [reply service/src/resources/hive-webapps/hiveserver2/logconf.jsp]: Yeah, it's a typo. The text should be "unless user user_name has admin privileges"

**Review by yongzhi — APPROVED** [review]

**Pull request merged**